vividus-plugin-rest-api/src/main/java/org/vividus/bdd/steps/api/JsonResponseValidationSteps.java

```diff
@@ -371,13 +379,23 @@ public void performAllStepsForProvidedJsonIfFound(ComparisonRule comparisonRule,
 
     private int getElementsNumber(String json, String jsonPath)
     {
-        Optional<List<?>> elements = getElements(json, jsonPath);
```

Comment:
it looks like all the changes below are not required anymore

Reply:
Okay.